packages/dev/s2-docs/pages/react-aria/GridList.mdx

```diff
@@ -27,52 +27,52 @@ export const description = 'Displays a list of interactive items, with support f
 
   <GridList/* PROPS */>
     <GridListItem textValue="Desert Sunset">
-      <img src="https://images.unsplash.com/photo-1705034598432-1694e203cdf3?q=80&w=600&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D" width={600} height={400} />
+      <img src="https://images.unsplash.com/photo-1705034598432-1694e203cdf3?q=80&w=600&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D" width={600} height={400} alt="Dessert Sunset" />
```

Comment:
I think we don't need alt on the images because the Text already has the label. So we just need to set `alt=""` to mark them as decorative.